bob/bob.go

```diff
@@ -1,15 +1,37 @@
-// This is a "stub" file.  It's a little start on your solution.
-// It's not a complete solution though; you have to write some code.
-
-// Package bob should have a package comment that summarizes what it's about.
-// https://golang.org/doc/effective_go.html#commentary
 package bob
 
-// Hey should have a comment documenting it.
+import (
+	"regexp"
+	"strings"
+)
+
+// Hey function which responds correctly
 func Hey(remark string) string {
-	// Write some code here to pass the test suite.
-	// Then remove all the stock comments.
-	// They're here to help you get started but they only clutter a finished solution.
-	// If you leave them in, reviewers may protest!
-	return ""
-}
+	remark = strings.Trim(remark, " \t\n\r")
```

Comment:
You can actually use the `strings.TrimSpace` function here. `TrimSpace` determines the white space characters by iterating over the string looking for ASCII non-space characters. What this means is that it treats many things as whitespace, including ` `, `\t`, `\n` and `\r`

Reply:
Oh wow thats awesome.